packages/flutter/lib/src/widgets/_platform_selectable_region_context_menu_web.dart

```diff
@@ -135,6 +135,9 @@ class PlatformSelectableRegionContextMenu extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Stack(
+      // Setting fit to StackFit.passthrough will prevent the child from
+      // misaligning on web and be consistent with other platforms.
+      fit: StackFit.passthrough,
       alignment: Alignment.center,
```

Comment:
I don't have a particular reason. I think I copied the code from pointerinterceptor or somwhere

Reply:
I think we should remove this alignment then instead of adding an explicit `Stack.fit`. I did some digging to see where this alignment originated in `pointer_interceptor` but didn't find a reason besides this commit when it was added [flutter/packages@`6adde6f` (#256)](https://github.com/flutter/packages/pull/256/commits/6adde6fe71ccf556cebd5bd22a2d003e4ca103d0) . Also talked to @ditman and removing the alignment didn't seem to have any adverse effects in our tests. The alignment is only affecting the position of the child widget since the `HTMLElementView` is wrapped with a `Positioned`.